crates/figma_import/src/document.rs

```diff
@@ -193,20 +203,26 @@ impl Document {
     pub fn new_if_changed(
         api_key: &str,
         document_id: String,
+        requested_version_id: String,
         proxy_config: &ProxyConfig,
         last_modified: String,
-        version: String,
+        last_version: String,
         image_session: Option<ImageContextSession>,
     ) -> Result<Option<Document>, Error> {
-        let document_head_url = format!("{}{}?depth=1", BASE_FILE_URL, document_id);
+        let mut document_head_url = format!("{}{}?depth=1", BASE_FILE_URL, document_id);
+        if requested_version_id.is_empty() == false {
+            document_head_url.push_str("&version=");
+            document_head_url.push_str(&requested_version_id);
+        }
         let document_head: FileHeadResponse =
             serde_json::from_str(http_fetch(api_key, document_head_url, proxy_config)?.as_str())?;
 
-        if document_head.last_modified == last_modified && document_head.version == version {
+        if document_head.last_modified == last_modified && document_head.version == last_version {
```

Comment:
If we request a specific version of the Figma file, when would last_modified be different, or document_head.version be different than last_version? I'm a little confused here.

Reply:
Actually it would not. But there could be cases that no local files saved and app loads the figma doc with a version for the first time.

Reply:
Yes, that is right.